cpp/util/json_wrapper.h

```diff
@@ -101,7 +101,7 @@ class JsonObject {
   }
 
   const char* ToString() const {
-    return json_object_to_json_string(obj_);
+    return json_object_to_json_string_ext(obj_, JSON_C_TO_STRING_PLAIN);
```

Comment:
I suggest we keep the change as a step toward better cleanliness, and keeping in line with what other libraries do (I'm thinking of the Go library).